src/analysis/multiSpecies/Microbiome_Modeling_Toolbox/PairwiseInteractionModeling/useDiet.m

```diff
@@ -0,0 +1,31 @@
+function [modelOut] = useDiet(modelIn, dietConstraints)
+% Implements diet constraints in a COBRA model structure.
+% USAGE:
+%
+%    [modelOut] = useDiet(modelIn, dietConstraints)
+%
+% INPUTS:
+%    modelIn:     original model
+%    dietConstraints:        cell array of three columns containing 
+% exchanges, lower bounds, and  upper bounds respectively
+%
+% OUTPUT:
+%    modelOut:    model with applied constraints
+
+% .. Authors:
+%       - Almut Heinken 16.03.2017
+%       - Laurent Heirendt March 2017
+%       - Almut Heinken 02/2018: Generalized script for any dietary
+%       constraints as input.
+
+model = modelIn;
+model = changeRxnBounds(model, model.rxns(strmatch('EX_', model.rxns)), 0, 'l');
```

Comment:
Whats the intention here? To set the lower bound of all reactions that allow uptake to 0 ? If, I would suggest using `findSExRxnInd` to determine the reactions. 
In any instance, the documentation should state which reactions will have their lower bound altered.